itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestActivePassiveHA.java

```diff
@@ -139,6 +139,72 @@ private static void setHAConfigs(Configuration conf) {
     conf.setInt(ConfVars.HIVE_ZOOKEEPER_CONNECTION_MAX_RETRIES.varname, 1);
   }
 
+  @Test(timeout = 60000)
+  public void testHealthCheck() throws Exception {
+    String instanceId1 = UUID.randomUUID().toString();
+    miniHS2_1.start(getConfOverlay(instanceId1));
```

Comment:
do we need to start miniHS2 to do the port checks?

Reply:
Yes, the test needs to bring up HS2 in Active-Passive-HA mode and check that the health check endpoint can be reached only via the dedicated port and not via the WebUI port. Other Active-Passive-HA tests also use MiniHS2